### PR TITLE
Add tests for bundle to verify the support for empty array and map

### PR DESCRIPTION
**Pull request by cherylEnkidu**
#no-changelog

### CODE REVIEW

**Review by wu-hui — APPROVED** [review]

File: packages/firestore/test/unit/util/bundle_data.ts [hunk 1]
```diff
@@ -195,7 +195,12 @@ export const doc2: BundleElement = {
     name: 'projects/test-project/databases/(default)/documents/collectionId/doc2',
     createTime: { seconds: 1, nanos: 2000000 },
     updateTime: { seconds: 3, nanos: 4000 },
-    fields: { foo: { stringValue: 'value1' }, bar: { integerValue: 42 } }
+    fields: {
+      foo: { stringValue: 'value1' },
+      bar: { integerValue: 42 },
+      emptyArray: { arrayValue: {} },
```
Comment (dconeybe) [inline packages/firestore/test/unit/util/bundle_data.ts:201]: Is `{}` supposed to be `[]` for `emptyArray`?

Comment (cherylEnkidu) [reply packages/firestore/test/unit/util/bundle_data.ts:201]: This syntax is coming from bundle.txt, in the bundle file, array object is represented as {}

**Review by dconeybe — APPROVED** [review]
